### ISSUE TITLE
disabled & aria-disabled validation test cases

### ISSUE THREAD

**Issue by scottaohara** [validator/validator#1106]
per these [test cases](https://w3c.github.io/html-aria/tests/disabled-test.html), would expect the validator to report the following:

test 1: results are largely as expected - validator throws no warnings/errors.  Exception should be that `input type=hidden aria-disabled=true"` should throw an error as no aria-* attributes are allowed on this input type.  (however, as this element is not exposed to users, I can understand if this is a "wontfix" issue.)

test 2: flags warnings as expected and needs no updates.

test 3: *nothing presently reported by validator* - should report errors for use of `aria-disabled=false` w/ `disabled` attribute, as they are in opposition and `disabled` attribute wins.

**Comment by sideshowbarker** [comment]
> per these [test cases](https://w3c.github.io/html-aria/tests/disabled-test.html), would expect the validator to report the following:
> 
> test 1: results are largely as expected - validator throws no warnings/errors. Exception should be that `input type=hidden aria-disabled=true"` should throw an error as no aria-* attributes are allowed on this input type. (however, as this element is not exposed to users, I can understand if this is a "wontfix" issue.)

Fixed by https://github.com/validator/validator/commit/84bde80

> test 3: _nothing presently reported by validator_ - should report errors for use of `aria-disabled=false` w/ `disabled` attribute, as they are in opposition and `disabled` attribute wins.

Fixed by https://github.com/validator/validator/commit/726e759
